packages/bot-web-ui/src/components/dashboard/bot-builder/toolbar/workspace-group.tsx

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+import { isMobile } from '@deriv/shared';
```

Comment:
This `isMobile` is going to be deprecated. Can we take the `isMobile` from the UI store?

Reply:
@shafin-deriv Sure👍, this PR is overdue, it needs to be rewritten, the codebase has changed significantly over the 3 sprints

Reply:
✅improved